<launch>
	
	<node pkg="object_perception" type="findingCenter" name="findingCenter" output="screen">
        <param name="FOCAL_LENGTH_X" value="1416.07692"/>
        <param name="FOCAL_LENGTH_Y" value="1420.90281"/>

<!--
        <param name="TUNED_H_DISTANCE_TOP_LEFT" value="-35.0"/>
        <param name="TUNED_V_DISTANCE_TOP_LEFT" value="20.0"/>

        <param name="TUNED_H_DISTANCE_BOTTOM_RIGHT" value="25.0"/>
        <param name="TUNED_V_DISTANCE_BOTTOM_RIGHT" value="70.0"/>
-->

<!-- webcam resolution 640x480
        <param name="TUNED_H_DISTANCE_TOP_LEFT" value="-80"/>
        <param name="TUNED_V_DISTANCE_TOP_LEFT" value="60"/>

        <param name="TUNED_H_DISTANCE_BOTTOM_RIGHT" value="-35"/>
        <param name="TUNED_V_DISTANCE_BOTTOM_RIGHT" value="110"/>
-->


        <param name="TUNED_H_DISTANCE_TOP_LEFT" value="-50"/>
        <param name="TUNED_V_DISTANCE_TOP_LEFT" value="100"/>

        <param name="TUNED_H_DISTANCE_BOTTOM_RIGHT" value="75"/>
        <param name="TUNED_V_DISTANCE_BOTTOM_RIGHT" value="250"/>


        <param name="ARM_RAIDUS" value="1.1"/>
    </node>

</launch>
